packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -5471,16 +5472,18 @@ void main() {
     log.clear();
 
     final EditableTextState state = tester.firstState(find.byType(EditableText));
-
     // setEditingState is not called when only the remote changes
-    state.updateEditingValue(const TextEditingValue(
+    state.updateEditingValue(TextEditingValue(
       text: 'a',
+      selection: controller.selection,
```

Comment:
if the selection is not set, it will be set to collapsed at -1, which should cause a selection change. This is not needed before because it previously just ignored the selection change